opentracing-impl-java8/src/test/java/io/opentracing/AbstractTracerTest.java

```diff
@@ -64,8 +58,8 @@ public SpanContext context() {
             }
         };
         Map<String,String> map = new HashMap<>();
-        TextMapWriter carrier = new TextMapImpl(map);
-        instance.inject(span.context(), carrier);
+        TextMap carrier = new TextMapImpl(map);
+        instance.inject(span.context(), Format.Builtin.TEXT_MAP, carrier);
```

Comment:
This is approximately how vanilla OpenTracing user code would look after this change. I personally prefer how clear/explicit this is, as inject/extract can be confusing and the in-place documentation of TEXT_MAP is useful. It also allows the compiler/IDE to force the format and carrier to match from a type-safety perspective (unlike `master` with its Object params).